### PR TITLE
Service Love + Economy

### PR DESCRIPTION
**Pull request by Kurfursten**
The successor to Chef Love PR, this adds a bunch of new service content to make the department more fun. It also has some economy stuff, because Service will never be satisfying without nerfing food costs. So @drcelt @deitylink @zth--

**Economics**
Prices at Getmore and Dan have been standardized, value based on nutrition. Getmore prices were raised somewhat, Dan prices lowered somewhat. Pizza from supply is now much more expensive.
- Getmore foods are now valued at price = 66% \* nutrition value (e.g.: SOS Jerky has 6u nutriment - 90 nutrition, so it costs $60). Syndiecakes remain free. Bustanuts remain free, but no longer contain nutriment.
- Dan foods are valued at 16.5% \* nutrition value. Overall, they became cheaper.
- Ramen is much cheaper than its nutritional value (10% of nutrition for Getmore ramen - $15 for 150, 2% of nutrition for Dan ramen - $2 for 100), but it has limited stock and in the case of Getmore ramen, needs you to add water.
- Cargo Pizza now costs $15 per pizza (instead of $15 total), so a full pack of five costs $75. (Together, they will provide 150-250 nutriment, a value of ~$2000)
- You can now hack Discount Dan machines to buy antitox pills.

**Chef**
- Carp and Spider dishes no longer provide bonus poison when cooked (but will still inherit it unless poison sacs are removed)
- Using the processor one can butcher out the poison sacs in poisonous foods (spider legs, spider meat, carp meat). This separates the poison which can then be used elsewhere, but purifies the food. Spider toxin sacs: toxicsac, Carp toxic spines: toxicspine
- Carp dishes now provide pheromones as a bonus
- Flour sacks contain 50u by default instead of 30u. This buffs the starting flour and basic food crate.
- coleslaw Coleslaw: 1 carrot, 1 cabbage, 2 vinegar
- honeyflan Honey Flan: 5u honey, 5u cinnamon, 5u milk, 1 egg
- riceball Rice ball: 5u rice
- risotto Risotto: 10u rice, 5u wine, 1 cheese wedge
- sushi Spicy Eggplant Sushi Rolls: 1 eggplant, 1 chili pepper, 10u rice, 2 vinegar
- tendies Chicken Fillet: 1 raw chicken, 3u corn oil
- PDA reagent scanner now shows exact units of each, in order to help chefs learn which meal provides the best nutriment payout.
- Firing food from a pneumatic cannon at someone will cause them to take a bite if it hits.

**Bartender**
- Hacking the Boozeomat now opens up coffee and mug options as well.
- Purchasing tea anywhere (hot drinks machine, hacked boozeomat) will now spawn one of three thrilling varieties, allowing you to get green and red tea without a cargo crate.
- Vending delay removed from the Boozeomat (it has no animation)
- pinacolada Pina Colada: 2 parts rum, 1 part cream, 1 part ice

### CODE REVIEW

**Comment by Kurfursten** [comment]
Although Travis has failed, this code does compile.

`vgstation13.dmb - 0 errors, 0 warnings (12/12/15 6:06 am)`

**Comment by DeityLink** [comment]
> foods are now valued at price = 66% \* nutrition value

This is long overdue.

**Comment by Arthurdentist** [comment]
> inject food with chloral / advanced slime toxin
> fire food at unsuspecting victim with pneumatic cannon
> enjoy your new sleepy customer / angry slime

Fun stuff, you even added more uses for vinegar. I endorse this PR.

**Comment by ghost** [comment]
> piece of jerky now costs more than a particle accelerator with singularity generator combined

The cost changes are irrational.

The reason for chef being irrelevant is because starvation is not fatal and everyone just becomes an anorexic bag of skin and bones functionally normally.

Also this absolutely inconsistent with everything else in the game

> if(poisonsacs && W.is_sharp() > 0.8 && user.mind.assigned_role == "Chef") //Only someone with a culinary background

This is not a character-skill based game, people can get job transfers and fully do their job. Except chaplain being a special snowflake for cult balance.

**Comment by Kurfursten** [comment]
@TermedClepe Collectable hats are the same price as 20 singularity generators.

The cost of a Getmore is premium because you know that no one else has handled the food. And remember, if you don't like the high price of Getmore you could...

> Eat early and snag a ramen, they're much cheaper than their nutrition output.
> Buy from Dan. You'll get annoying messages unless you take antitoxin (now more widely available), but it's 4x cheaper relative to nutrition output.
> Order Pizza. You have to wait on Cargo, but it's 26.4x cheaper relative to nutrition output.
> Hack the machine. It will spit out food for free.
> Hack the machine for contraband and eat Syndie Cakes, which are still free.
> Get free food from Kitchen and Botany.

Discouraging people to live their lives entirely on Getmore helps increase the relevance of the kitchen. With all these alternative options, I'm not too worried that people will starve.

**Comment by ghost** [comment]
You're forgetting even with current low vendor and pizza costs, people are completely fine with going entire round without eating a speck of food because the penalties are irrelevant to them.

**Comment by Kurfursten** [comment]
The vending price tweaks won't affect those people, and that's okay.

**Comment by ghost** [comment]
Also, refer to the "chef culinary skill" issue, this is not a character-skill based game and all other jobs can fully function without starting as their job with exception of chaplain for game mode balance reasons.

**Comment by Kurfursten** [comment]
It's something to discuss, sure. Although you are forgetting the bartender, clown, and mime in addition to the chaplain.

**Comment by ghost** [comment]
Mime is mute. This is not a skill requirement for anything.
Clown is clumsy. This is a mutation anyone can get and can be removed from clown.
Chaplain is a direct counter to cult and vampires (and wizard jaunts?). If it was applicable to regular crew it would completely destroy these antagonists.

What does bartender have? I'm guessing iron liver, which is again a mutation.

**Comment by Kurfursten** [comment]
The Mime has the force wall spell which is completely unobtainable. And saying "anyone can get" the mutations associated with those jobs is oversimplifying genetics a bit. It's like saying anyone can get a supermatter shard. Sure, if they have an emitter and access to a cargo ID.

But I see you're not going to be convinced - so I have your opinion at least. If there's consensus with you, I'll take it out. But I do like the idea that there is something food-related that the crew would actually go to the chef for. Especially because cutting the poison out of meat is in real life a culinary feat (e.g.: fugu).

**Comment by ghost** [comment]
Forgot about the mime wall.

Chef has all the food machines already, tying food specifically to a chefs mind is silly when everyone can do heart surgery.

**Comment by d3athrow** [comment]
Fix your changelog

**Comment by Zth--** [comment]
I love this. Btw, why a supermatter shard would be so cheap? Raise its value to 2000 or 3000. Even an assistant can buy one

**Comment by FudgePucker** [comment]
> Anyone can do brain surgery and set up a super matter engine, but god forbid if anybody but the chef tries to remove a toxin sac from a fish

:-1:, people who change jobs get a kick in the balls from this. It also makes zero sense from a balance point of view. And arguing the slippery slope argument, if only the chef can remove toxin sacs, then why can't everyone have the inability to make machines besides the mechanic? why can't everyone have the inability to see what is in a beaker besides the bartender and chemists? why can't everyone have the inability to wrench down pipes besides engineers? why can't everyone have the inability to do surgery besides medical doctors? why can't everyone have the inability to handcuff people besides security officers?

No real class check has been implemented, why start with something as silly as removing toxin sacs?

Other than that :+1:

**Comment by Kurfursten** [comment]
@Zth-- There's really no reason that you couldn't put the shard at that level of expense given that the CE should be ordering it (it has CE access) and he has the department account with $5000. That being said, this PR is just focused on food

@PJB3005 As mentioned in the first reply, this does compile

**Comment by Kurfursten** [comment]
@FudgePucker Fairly argued. Maybe I'll move it to the food processor or something.

**Comment by FudgePucker** [comment]
@Kurfursten 
Isn't there that nice fancy butcher menu?
Why not have it so that you can remove the glands and sacs from there?

**Comment by Kurfursten** [comment]
They're contained within the meat, not within the mob. This is so you can still cook the old poisonous versions of the dishes by not removing the poison sacs.

**Comment by FudgePucker** [comment]
@Kurfursten This might be hard to implement, but what if they don't remove the toxin gland first from the mob, then when they butcher for meat it causes it to get into it. That way they can still have the old poison meals.

If you want to balance it so that you can't get free nutrient with just a glass shard, make it so that you cut into the sac and it bleeds into the meat if you do it without a knife or scalpel.

**Comment by PJB3005** [comment]
@Kurfursten I'll let you figure out why it's not compiling.

And for the record, it's NOT just Travis fucking up.

**Comment by Kurfursten** [comment]
That sounds passive aggressive

**Comment by 9600bauds** [comment]
@Kurfursten changelogs are now Travis-enforced, which means they can fail compilation. 

I completely disagree with PJB's logic of "I'll let you figure it out" because the problems that can make them NOT compile ARE NOT STIPULATED ANYWHERE AND THEY SHOULD KIND OF MAYBE HAVE A WARNING IN THE EXAMPLE CHANGELOG.

You need to put all the lines that contain a ":" inbetween quotes in your changelog, and you have to put \ behind every quotation mark besides at the start and at the end.
Maybe other shit too, WHO KNOWS

**Comment by Kurfursten** [comment]
Thanks.

**Comment by Exxion** [comment]
> Bartender changes

THANK YOU THAT WAS ALL VERY ANNOYING

**Comment by FudgePucker** [comment]
Coconuts and pine apples when?
Using a knife on them makes fancy drinking glasses.

**Comment by Kurfursten** [comment]
When someone sprites both trees complete with multiple stages of growth + sprites both products + cut versions

And even then it would involve writing seeds, fruits, plus pineapple juice certainly. Grinding coconuts for milk would be neat though.

**Comment by Kurfursten** [comment]
@PJB3005 No longer compile failure

**Comment by SangriaMonkey** [comment]
> tendies

:skull: :gun:

**Comment by ghost** [comment]
Removing bustanuts nutriments ? How devilish now that hardcore mode can also be enabled

At least the crew will be more diligent about getting a Chef in place, I suppose

**Comment by unid15** [comment]
> Whoever proposes a PR [[with chicken tendies]] and whoever merges it is getting banned from the repo

rip in peace @Kurfursten

File: code/modules/food/recipes_microwave.dm [hunk 1]
```diff
@@ -1397,6 +1413,11 @@
 		)
 	result = /obj/item/weapon/storage/fancy/food_box/chicken_bucket
 
+/datum/recipe/chicken_tendies
```
Comment (d3athrow) [inline code/modules/food/recipes_microwave.dm]: ban ban ban ban BAN BAN BANBANBANBANBANBAN

**Comment by Kurfursten** [comment]
I just asked the thread if anyone had any food sprites they wanted added and was provided those six (plus one drink).

**Comment by d3athrow** [comment]
remove tendies
e
m
o
v
e
t
e
n
d
i
e
s

**Comment by Kurfursten** [comment]
Very well, I needed to fix conflicts anyway

**Comment by Kurfursten** [comment]
No more tendies, no more conflicts, no more compile fail, and squashed. Good to go.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -3246,6 +3236,17 @@
 	reagents.add_reagent("nutriment", 3)
 	bitesize = 1
 
+/obj/item/weapon/reagent_containers/food/snacks/chicken_fillet
+	name = "Chicken Fillet"
+	desc = "This is a fancy word for chicken fingers so that high class people can forget they're eating fried food."
+	icon_state = "tendies"
```
Comment (d3athrow) [inline code/modules/reagents/reagent_containers/food/snacks.dm:3242]: triggered

**Pull request merged**